### PR TITLE
Adapted my work on Orders to the current iteration of Dev

### PR DESCRIPTION
**Pull request by travispinkerton**
Fixed the orders that are rendered so they only pertain to the user, if they are an Admin. Also adjusted the logout functionality and a couple other minor changes, working smoothly and properly now! Minor adjustments will be needed to maintain styling cohesiveness, this is just an MVP for now.

### CODE REVIEW

File: routes/orders.js [hunk 1]
```diff
@@ -3,10 +3,10 @@ const { getAllOrders, getCartByUser, createOrder } = require('../db/orders');
 const { getOrderProductById, addProductToOrder, updateOrderProduct } = require('../db/order_products');
 const { requireUser, requireAdmin } = require('./utils');
 
-ordersRouter.get('/', requireAdmin, async (req, res, next) => {
```
Comment (albert4770) [inline routes/orders.js:6]: Why did you remove the requireAdmin middleware?

File: routes/users.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ usersRouter.post('/register', async (req, res, next) => {
 				user: newUser.username,
 				isAdmin: newUser.isAdmin
 			},
-			JWT_SECRET
+			process.env.JWT_SECRET
```
Comment (albert4770) [inline routes/users.js:36]: The secret was already destructured up top

**Review by albert4770 — COMMENTED** [review]
Looks good, just curious about the changes to orders router

File: routes/orders.js (hunk 1 above)
Comment (travispinkerton) [reply routes/orders.js:6]: That was actually an oversight! I need to add that back in, good catch. I think I had taken it out for testing purposes and just neglected to put it back.

File: routes/users.js (hunk 2 above)
Comment (travispinkerton) [reply routes/users.js:36]: I know, I saw that and tried to keep it as is, but for some reason I kept getting a "missingsecretkey" error when I kept it as is. I just changed it to circumvent that error, which it did. We maybe be able to change it back no problem, not sure what the origin of my problem with that piece was still. Good catch as well!

**Review by Kwalsh2379 — APPROVED** [review]
Going to make the changes on my PR right after this
